sass/_base.scss

```diff
@@ -294,3 +294,39 @@ a>code {
         filter: invert(1);
     }
 }
+
+.youtube_placeholder div {
+    display: flex;
+    flex-direction: column;
+    background-color: #333;
+    color: #fff;
+    padding: 1.5rem;
+    gap: 1.5rem;
+
+    img {
+        margin-inline: auto;
+    }
+
+    p {
+        max-width: 30rem;
+        text-align: center;
+        margin-inline: auto;
+    }
+
+    button {
+        max-width: fit-content;
+        margin-inline: auto;
+        padding: .5rem 1rem;
+        border-radius: 9999px;
+        background-color: #fff;
+        border: 1px solid transparent;
+        color: #000;
+        cursor: pointer;
+
+        &:hover {
+            border: 1px solid #fff;
+            color: #fff;
+            background-color: #333;
+        }
+    }
+}
```

Comment:
```suggestion

.youtube_placeholder {
    display: flex;
    flex-direction: column;
    justify-content: center;
    aspect-ratio: 16/9;
    width: 100%;
    background-color: #333;

    div {
        display: flex;
        flex-direction: column;
        color: #fff;
        padding: 1.5rem;
        gap: 1.5rem;

        img {
            margin-inline: auto;
        }

        p {
            max-width: 30rem;
            text-align: center;
            margin-inline: auto;
        }

        button {
            max-width: fit-content;
            margin-inline: auto;
            padding: .5rem 1rem;
            border-radius: 9999px;
            background-color: #fff;
            border: 1px solid transparent;
            color: #000;
            cursor: pointer;

            &:hover {
                border: 1px solid #fff;
                color: #fff;
                background-color: #333;
            }
        }
    }
}
```

![image](https://github.com/matrix-org/matrix.org/assets/1374914/ae33f05f-67e6-4cd6-bb3a-ef15a06f257a)
![image](https://github.com/matrix-org/matrix.org/assets/1374914/f1317c5f-b20a-4834-908b-6ec2e59245e6)

I dont think we can as close on mobile as on desktop due to the required text space. But this at least is closer to what the iframe does.